src/tkn.ts

```diff
@@ -551,6 +552,12 @@ function compareTimeNewestFirst(a: TektonNode, b: TektonNode): number {
     return aTime < bTime ? 1 : -1;
 }
 
+function getStderrString(data: string | Error): string {
+    if (!(data instanceof Error)) {
+        return data;
+    }
+}
+
```

Comment:
It is better to rewrite this function in :
```typescript
function getStderrString(data: string | Error | undefined): string | undefined {
    if (data instanceof Error) {
        return data.message;
    } else if ((typeof data === 'string')) {
        return data;
    }
}
```
Because  in current implementation you may loose error message from `Error` type, and it always better to mark that function can return `undefined`.